teeio-validator/library/helperlib/teeio_common.c

```diff
@@ -0,0 +1,76 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2023-2024 Intel. All rights reserved.
+ *  License: BSD 3-Clause License.
+ **/
+
+#include <stdint.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+#include "pcie.h"
+#include "ide_test.h"
+#include "teeio_debug.h"
+#include "helperlib.h"
+#include "helper_internal.h"
+
+extern ide_run_test_group_result_t* g_current_group_result;
+
+bool teeio_record_assertion_result(
+  int case_class,
+  int case_id,
+  int assertion_id,
+  IDE_COMMON_TEST_CASE_ASSERTION_TYPE assertion_type,
+  IDE_COMMON_TEST_CASE_ASSERTION_RESULT assertion_result,
+  const char* extra_data)
```

Comment:
in SPDM-Responder-Validator, the last param is 
```
    const char *message_format,
    ...);
```

Reply:
Updated.